lib/zipkin-tracer/hostname_resolver.rb

```diff
@@ -28,24 +28,19 @@ def resolved_ip_address?(ip_string)
         ip_string.to_i.to_s == ip_string
     end
 
-    def each_annotation(spans, &block)
+    def each_endpoint(spans, &block)
```

Comment:
maybe add a comment about what this is doing? I'm guessing it is lazy parsing?

Reply:
I think this comment is explaining:
https://github.com/openzipkin/zipkin-ruby/blob/f0435f983ca63644db0fdf911f5bb52b2b9df87b/lib/zipkin-tracer/hostname_resolver.rb#L5-L6